changelog.d/14046.misc

```diff
@@ -0,0 +1 @@
+Fix CI config for building wheels on OSX. Regressed in #13983.
```

Comment:
Should probably be merged with the changelog in #13983?

Reply:
huh, I certainly meant to do this but couldn't find it. I'd assumed it  had made its way into the RC---but seemingly not. Will amend.